src/pages/JobSearch/index.jsx

```diff
@@ -53,6 +60,12 @@ function JobSearch() {
   // This is used to determine next result items to show when the user clicks "Load More" button.
   const [cursor, setCursor] = useState(12);
   const [jobs, setJobs] = useState(jobsData.slice(0, cursor));
+  console.log(cursor);
```

Comment:
Can you please remove this `console.log`? Thank you!

Reply:
Oops! I'm sorry I totally forgot to remove it.